.gitignore

```diff
@@ -3,5 +3,10 @@
 /examples/php/vendor
 /.devbox
 /gradle.properties
+/vendor
```

Comment:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/ :trollface:

Reply:
I have it for .DS_Store and a few things more. I thought that vendor wasn't one of them.